Resources/doc/internals/configuring_the_http_client.md

```diff
@@ -27,4 +27,93 @@ hwi_oauth:
 
 If you don't want to use the HTTPlug Bundle, the client and factory services **must be specified**.
 
+## Difference since 0.6+ version
+
+On v0.5 and below, you was able to configure your http client like that:
+
+```yaml
+# app/config/config.yml
+
+hwi_oauth:
+    http_client:
+        timeout:       10 # Time in seconds, after library will shutdown request, by default: 5
+        verify_peer:   false # Setting allowing you to turn off SSL verification, by default: true
+        ignore_errors: false # Setting allowing you to easier debug request errors, by default: true
+        max_redirects: 1 # Number of HTTP redirection request after which library will shutdown request,
+                         # by default: 5
+        proxy: "example.com:8080" # String with proxy configuration for cURL connections, ignored by default.
+                                    # "" -> don't set proxy and will use proxy system
+                                    # "example.com:8080" -> set custom proxy
+                                    # ":" -> disable proxy usage, ignoring also proxy system and ENVIRONMENT variables
+```
+
+As we not use Httplug, the http configuration does not rely from our bundle anymore.
```

Comment:
As we don't...

Reply:
> does not rely from our bundle anymore

Not sure what you mean here. Whose bundle? Does not rely on HttplugBundle or HWIOAuthBundle?

Reply:
@sagikazarmark It was "now", not "not". :wink: